test-worker/test/asset-compute/worker/png-webp-resize/params.json

```diff
@@ -0,0 +1,4 @@
+{
+    "width": 319,
+    "height": 319
+}
```

Comment:
maybe I am missing something, where is the fmt param telling us to produce a webp rendition?

Reply:
(It can be implicit if the filename has the webp extension, courtesy of imagemagick)

Reply:
The expected rendition file is named `rendition.webp` and `test-worker` [uses that extension](https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/lib/testrunner.js#L292-L295) if it's not in the `params.json`. This makes it easier for cases without extra params, as you don't need a `params.json` file at all.

Reply:
nice job test-worker!